lib/Room.php

```diff
@@ -726,6 +726,30 @@ public function getActiveSessions() {
 		return $sessions;
 	}
 
+	public function getUserIds($inCall = false) {
+        $query = $this->db->getQueryBuilder();
+        $query->select('user_id')
+            ->from('talk_participants')
+            ->where($query->expr()->eq('room_id', $query->createNamedParameter($this->getId(), IQueryBuilder::PARAM_INT)))
+            ->andWhere($query->expr()->nonEmptyString('user_id'));
+
+        if ($inCall) {
+            $query->andWhere($query->expr()->eq('in_call', $query->createNamedParameter(1, IQueryBuilder::PARAM_INT)));
```

Comment:
Could you please change this to
```
$query->andWhere($query->expr()->neq('in_call', $query->createNamedParameter(0, IQueryBuilder::PARAM_INT)));
```

This will later already support the change from #914.

Reply:
Good catch

Reply:
Done!